packages/ra-ui-materialui/src/layout/Error.tsx

```diff
@@ -100,12 +100,37 @@ const Error = props => {
     );
 };
 
-Error.propTypes = {
-    classes: PropTypes.object,
+export declare type ErrorClassKey =
+    | 'container'
+    | 'icon'
+    | 'toolbar'
+    | 'title'
+    | 'panel'
+    | 'panelDetails';
+
+export interface ErrorProps {
+    classes?: Record<ErrorClassKey, string>;
+    className?: string;
+    error: Error;
+    errorInfo?: Record<'componentStack', string>;
```

Comment:
where does that come from?

Reply:
It comes from https://github.com/marmelab/react-admin/blob/next/packages/ra-ui-materialui/src/layout/Layout.js#L134